indico/modules/events/surveys/templates/submission_preview.html

```diff
@@ -0,0 +1,69 @@
+{% extends 'layout/base.html' %}
+
+{% block title %}
+    {% trans %}Survey{% endtrans %}
+{% endblock %}
+
+{% block subtitle %}
+    {% trans %}Preview of the submission{% endtrans %}
+{% endblock %}
+
+{% block content %}
+    <div class="page-content-description">
+        {% set submission_dt = submission.submitted_dt | format_datetime %}
+        <p>
+            {% if submission.user %}
+                {% trans submitter_name=submission.user.full_name %}
+                    Submitter: {{ submitter_name }}
+                {% endtrans %}
+            {% else %}
+                {% trans %}
```

Comment:
`{% trans -%}...{%- endtrans %}` to avoid leading/trailing whitespace in the extracted data